app/lib/frontend/email_sender.dart

```diff
@@ -41,7 +42,14 @@ class EmailSender {
     } else {
       _logger.info('Sending email: $debugHeader...');
       try {
-        await send(_toMessage(message), _server);
+        await retry(
+          () => send(
+            _toMessage(message),
+            _server,
+            timeout: Duration(seconds: 15),
+          ),
+          delayFactor: Duration(seconds: 2),
+        );
```

Comment:
Can we set `maxAttempts: 2` -- I don't want to risk people getting 7 duplicate emails :D